### PR TITLE
[CV Eval] CV Version Test

### PR DESCRIPTION
**Pull request by sambible**
Test for CV Version table, as well as for this CC BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1911545
(Reopening of an older PR)

Requires: https://github.com/SatelliteQE/airgun/pull/1083

### CODE REVIEW

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k test_version_table_read
airgun: 1083

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6788
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_contentview.py -k test_version_table_read --external-logging
Test Result : ========== 1 failed, 2 deselected, 16 warnings in 1073.05s (0:17:53) ===========
```

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k test_version_table_read
airgun: 1083

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6792
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_contentview.py -k test_version_table_read --external-logging
Test Result : ========== 1 failed, 2 deselected, 16 warnings in 1149.65s (0:19:09) ===========
```

File: tests/foreman/ui/test_contentview.py [hunk 1]
```diff
@@ -36,6 +38,62 @@ def test_positive_create_cv(session, target_sat):
...
+    cv.publish()
+    with target_sat.ui_session() as session:
+        package_name = 'aajohan'
+        session.organization.select(org_name=function_sca_manifest_org.name)
+        response = session.contentview_new.read_version_table(
+            cv.name, 'Version 1.0', 'rpmPackages', search_param=package_name
+        )
+        assert package_name in response[0]['Name']
+        assert response[0]['Epoch']
+        assert response[0]['Name'] == 'aajohan-comfortaa-fonts-3.001-2.el8.noarch'
```
Comment (shweta83) [inline tests/foreman/ui/test_contentview.py:92]: This test will fail on RHEL9 Satellite.

Comment (shweta83) [inline tests/foreman/ui/test_contentview.py:92]: Can we avoid the hardcoding of version?

File: tests/foreman/ui/test_contentview.py [hunk 2]
```diff
@@ -36,6 +38,62 @@ def test_positive_create_cv(session, target_sat):
...
+        assert response[0]['Version'] == '3.001'
```
Comment (shweta83) [inline tests/foreman/ui/test_contentview.py:93]: Same as above

File: tests/foreman/ui/test_contentview.py [hunk 3]
```diff
@@ -36,6 +38,62 @@ def test_positive_create_cv(session, target_sat):
...
+        assert response[0]['Release'] == '2.el8'
```
Comment (shweta83) [inline tests/foreman/ui/test_contentview.py:94]: Same as above

**Review by shweta83 — CHANGES_REQUESTED** [review]

**Review by vsedmik — COMMENTED** [review]
Just a small proposal.

File: tests/foreman/ui/test_contentview.py [hunk 4]
```diff
@@ -36,6 +38,62 @@ def test_positive_create_cv(session, target_sat):
...
+    rhel8_bos_info = target_sat.cli.RepositorySet.info(
+        {
+            'name': REPOS['rhel8_bos']['reposet'],
+            'organization-id': function_sca_manifest_org.id,
+            'product': REPOS['rhel8_bos']['product'],
+        }
+    )
+    rh_repo_id = rhel8_bos_info['enabled-repositories'][0]['id']
+    rh_repo = target_sat.api.Repository(id=rh_repo_id).read()
+    rh_repo.sync()
```
[the remark this answers is not in the document]
Comment (sambible) [reply tests/foreman/ui/test_contentview.py:80]: I can look at a rework to use a smaller repo, yeah. No real reason it has to be this.

File: tests/foreman/ui/test_contentview.py (hunk 3 above)
Comment (sambible) [reply tests/foreman/ui/test_contentview.py:94]: Yeah, I can grab these from the repo and make sure they aren't hardcoded.

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k test_version_table_read
airgun: 1083

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6836
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_contentview.py -k test_version_table_read --external-logging
Test Result : =========== 1 failed, 2 deselected, 16 warnings in 819.66s (0:13:39) ===========
```

File: tests/foreman/ui/test_contentview.py (hunk 3 above)
Comment (shweta83) [reply tests/foreman/ui/test_contentview.py]: Thank you.

**Review by vsedmik — APPROVED** [review]
ACK, pending PRT pass (seems like some airgun tweak is needed).

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k test_version_table_read
airgun: 1083

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6880
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_contentview.py -k test_version_table_read --external-logging
Test Result : =========== 1 failed, 2 deselected, 16 warnings in 850.71s (0:14:10) ===========
```

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k test_version_table_read
airgun: 1083

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6881
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_contentview.py -k test_version_table_read --external-logging
Test Result : =========== 1 passed, 2 deselected, 16 warnings in 920.91s (0:15:20) ===========
```

**Review by damoore044 — APPROVED** [review]

**Pull request merged**